DancingGoat/Views/Shared/_Layout.cshtml

```diff
@@ -15,25 +15,25 @@
                     <nav role="navigation">
                         <ul>
                             <li>
-                                @Html.ActionLink(Localizer["Home"], "Index", "Home")
+                                <a asp-controller="Home" asp-action="Index">@Localizer["Home"]</a>
                             </li>
                             <li>
-                                @Html.ActionLink(Localizer["Store"], "Index", "Coffees")
+                                <a asp-controller="Coffees" asp-action="Index">@Localizer["Store"]</a>
```

Comment:
This points to a wrong route in Spanish...it goes to https://localhost:5001/es-ES/coffees/indice which is 404